src/commands/publish-application.ts

```diff
@@ -88,8 +88,12 @@ async function installApplication(terminal:vscode.Terminal) {
 async function readCloudProfile() {
     var fs = require('fs');
     const cloudProfile: vscode.Uri[] = await vscode.workspace.findFiles('**/Cloud.json');
-    const pathToCloudProfile = cloudProfile[0].fsPath.replace('/c:', '');
+    if (cloudProfile.length < 1) {
```

Comment:
is this check present in deploy cmd ?

Reply:
In publish we check for the presence of Cloud.json, and In deploy we check for the presence of install.ps1 and return error messages accordingly. However, install.ps1 is created during application creation but cloud.json is created during application build time. 
So when deploy is run before build but after create, the commands in install.ps1 fail with 'could not find application packages' etc.